cherryontech/ywgc
# chore: add tailwind

## Work done
- installed tailwindcss and relevant files
- added some initial styling to ensure tailwind is working
- removed irrelevant react starter code

## Manual testing instructions

1. npm run start
- all page content should now be inside a gray div
- h1 for each page should be centered
